services/cincinnati-ci/buildconfig.yaml

```diff
@@ -0,0 +1,23 @@
+kind: BuildConfig
+apiVersion: v1
+metadata:
+  name: cincinnati-build-root-latest
+  namespace: cincinnati-ci
+  labels:
+    ci.openshift.io/source-repo: cincinnati
+spec:
+  successfulBuildsHistoryLimit: 1
+  failedBuildsHistoryLimit: 2
+  output:
+    to:
+      kind: ImageStreamTag
+      name: cincinnati-build-root:latest
+  source:
+    git:
+      uri: "https://github.com/openshift/cincinnati"
+      ref: master
+  strategy:
+    dockerStrategy:
+      dockerfilePath: dist/Dockerfile.build/Dockerfile
+  triggers:
+    - type: ConfigChange
```

Comment:
How would the rebuild be triggered, i.e. where is the config this refers to?

Reply:
This trigger would start the build when BuildConfig spec changes, not the file itself. 

Prow periodically syncs the definitions of this build with BuildConfig on the server, applying any changes. This definition is incomplete, so prow would trigger builds on every sync, so base image would be periodically rebuilt with no actions on our side.